docs/reference-guides/data/data-core.md

```diff
@@ -4,9 +4,106 @@ Namespace: `core`.
+
+### getPostType
+
+Returns the information for a given post type.
+
+_Usage_
+
+    import { useSelect } from '@wordpress/data';
+    import { store as coreDataStore } from '@wordpress/core-data';
+
+    const postType = useSelect(
+    	( select ) => select( coreDataStore ).getPostType( 'post' )
+
+    	// Equivalent to: select( coreDataStore ).getEntityRecord( 'root', 'postType', 'post' )
+    );
+
+_Parameters_
+
+-   postType `string`
+
+_Returns_
+
+-   `EntityRecord | undefined`: Record.
+
+### getPostTypes
+
+Returns the information for post types.
+
+_Usage_
+
+    import { useSelect } from '@wordpress/data';
+    import { store as coreDataStore } from '@wordpress/core-data';
+
+    const postTypes = useSelect( ( select ) => {
+    	return select( coreDataStore ).getPostTypes( { per_page: 4 } );
+
+    	// Equivalent to:
+    	// select( coreDataStore ).getEntityRecords( 'root', 'postType', { per_page: 4 } );
+    } );
+
+_Parameters_
+
+-   _query_ `GetRecordsHttpQuery`: Optional terms query. If requesting specific fields, fields must always include the ID. For valid query parameters see the [Reference](https://developer.wordpress.org/rest-api/reference/) in the REST API Handbook and select the entity kind. Then see the arguments available for "List [Entity kind]s".
+
+_Returns_
+
+-   `EntityRecord[] | null`: Records.
+
+### getTaxonomy
+
+Returns information for a given taxonomy.
+
+_Usage_
+
+    import { useSelect } from '@wordpress/data';
+    import { store as coreDataStore } from '@wordpress/core-data';
+
+    const taxonomy = useSelect( ( select ) => {
+    	return select( coreDataStore ).getTaxonomy( 'category' );
+
+    	// Equivalent to:
+    	// select( coreDataStore ).getEntityRecord( 'root', 'taxonomy', 'category' );
+    } );
+
+_Parameters_
+
+-   taxonomy `string`
+
+_Returns_
+
+-   `EntityRecord | undefined`: Record.
+
+### getTaxonomies
+
+Returns information for taxonomies.
+
+_Usage_
+
+    import { useSelect } from '@wordpress/data';
+    import { store as coreDataStore } from '@wordpress/core-data';
+
+    const taxonomies = useSelect( ( select ) => {
+    	return select( coreDataStore ).getTaxonomies( { type: 'post' } );
+
+    	// Equivalent to:
+    	// select( coreDataStore ).getEntityRecords( 'root', 'taxonomy', { type: 'post' } );
+    } );
+
+_Parameters_
+
+-   _query_ `GetRecordsHttpQuery`: Optional terms query. If requesting specific fields, fields must always include the ID. For valid query parameters see the [Reference](https://developer.wordpress.org/rest-api/reference/) in the REST API Handbook and select the entity kind. Then see the arguments available for "List [Entity kind]s".
+
+_Returns_
+
+-   `EntityRecord[] | null`: Records.
+
 <!-- START TOKEN(Autogenerated selectors|../../../packages/core-data/src/selectors.ts) -->
 
-### canUser
+#### canUser
```

Comment:
Should this have been changed to an `<h4>` (`####`)? It appears to be a sub-heading for **Selectors**, which is an `<h2>`, and the other sub-headings above are all `<h3>`s.

Also, same question for the other changes later. Otherwise, changes look good.

Reply:
Hmm. That seems odd. Those were all generated by doc gen. Let me look into it